MediaBrowser.Model/Configuration/EncodingOptions.cs

```diff
@@ -11,11 +11,11 @@ public class EncodingOptions
         /// <summary>
         /// FFmpeg path as set by the user via the UI
         /// </summary>
-        public string EncoderAppPathCustom { get; set; }
+        public string EncoderAppPath { get; set; }
```

Comment:
I had these the wrong way round, such that existing custom paths would have been lost in new software running these changes.  Now we retain the original name for storing the custom path and use a new 'Display' field for populating the UI transcode page.